libs/coin-evm/src/api/etherscan.ts

```diff
@@ -102,7 +102,28 @@ export const getLastTokenOperations = async (
     url,
   });
 
-  return ops.map((event, index) => etherscanERC20EventToOperations(accountId, event, index)).flat();
+  // Why this thing ?
```

Comment:
Nice comment, we could just add that event <=> operation is 1/1 correlation

Something like "We want one operation per event, but one transaction triggers multiple events, that's why ..."

Not important and can be ignored